chainerx_cc/chainerx/python/routines.cc

```diff
@@ -327,6 +327,15 @@ void InitChainerxIndexing(pybind11::module& m) {
 void InitChainerxLinalg(pybind11::module& m) {
     // linalg routines
     m.def("dot", [](const ArrayBodyPtr& a, const ArrayBodyPtr& b) { return MoveArrayBody(Dot(Array{a}, Array{b})); }, "a"_a, "b"_a);
+
+    pybind11::module mlinalg = m.def_submodule("linalg");
+    mlinalg.def("_is_lapack_available", []() -> bool { return CHAINERX_ENABLE_LAPACK; });
+    mlinalg.def(
+            "solve",
+            [](const ArrayBodyPtr& a, const ArrayBodyPtr& b) { return MoveArrayBody(Solve(Array{a}, Array{b})); },
+            py::arg("a"),
+            py::arg("b"));
```

Comment:
We've started to adapt the `"b"_a` literal syntax. Could you use it too? https://pybind11.readthedocs.io/en/stable/basics.html#keyword-arguments

Reply:
Done.